package.json

```diff
@@ -27,13 +27,15 @@
   "devDependencies": {
     "coveralls": "^3.1.0",
     "eslint": "^7.2.0",
+    "flat": "^5.0.2",
     "glob": "^7.1.6",
     "jest": "^26.0.1",
     "jest-cli": "^26.0.1",
     "lerna": "^3.22.0",
     "lodash": "^4.17.15",
     "markdown-magic": "^0.1.25",
     "moleculer": "^0.14.7",
+    "nedb": "^1.8.0",

```

Comment:
`nedb` & `flat` no need here because moleculer-db/package.json contains it where they are needed.